src/index.js

```diff
@@ -211,3 +211,98 @@ const validator = {
 };
 
 export default validator;
+
+export {
+  version,
```

Comment:
We'll need to update this every time we do any new validator?

Reply:
Yes. Or you can remove `const validator = { ... };` and `export default validator` to remove duplicities. It would be the best solution as `export default validator` will never be tree-shakable. But it would introduce breaking change.